packages/@react-spectrum/breadcrumbs/stories/Breadcrumbs.stories.tsx

```diff
@@ -136,6 +139,10 @@ storiesOf('Breadcrumbs', module)
         <Item>Root</Item>
       </Breadcrumbs>
     )
+  )
+  .add(
```

Comment:
are are we adding a new story in a PR to fix lint warnings?

Reply:
Yeah, just so we can test if anything breaks